test/integration/features/basic.feature

```diff
@@ -10,6 +10,42 @@ Feature: Basic
      Then Minishift should have state "Running"
       And Minishift should have a valid IP address
 
+  Scenario: OpenShift is ready after startup
+    After startup of Minishift OpenShift instance should respond correctly on its html endpoints
+    and OpenShift web console should be accessible.
+    Given Minishift has state "Running"
+     Then body of HTTP request to "OpenShift" at "/healthz" contains "ok"
+      And body of HTTP request to "OpenShift" at "/healthz/ready" contains "ok"
+      And body of HTTP request to "OpenShift" at "/console" contains "<title>OpenShift Web Console</title>"
+
+  Scenario Outline: User can set, get, view and unset values in configuration file
+    User is able to setup persistent configuration of Minishift using "config" command
+    and its subcommands, changing values stored in "config/config.json".
+    #To get values of nested keys, use following dot formating: key.nestedKey.
+    #If an array is expected, use following formating: [value1 value2 value3].
+    Given Minishift has state "Running"
+     When executing "minishift config set <property> <value>" succeeds
+     Then JSON config file "config/config.json" contains key "<property>" with value "<value>"
+      And stdout of command "minishift config get <property>" is equal to "<value>"
+      And stdout of command "minishift config view --format {{.ConfigKey}}:{{.ConfigValue}}" contains "<property>:<value>"
+     When executing "minishift config unset <property>" succeeds
+     Then stdout of command "minishift config get <property>" is equal to "<nil>"
+      And JSON config file "config/config.json" does not contain key "<property>"
+      
+  Examples: Config values to work with
+    | property  | value |
+    | disk-size | 22g   |
+    | memory    | 2222  |
+    | cpus      | 3     |
+
+  Scenario: Commands console and ip returns valid ip
+    User is able to get ip of Minishift VM with command "ip" and url adress of OpenShift instance
+    by running command "console" with flag "--url". Both commands should contain same IP.
+    Given Minishift has state "Running"
+     When executing "minishift console --url" succeeds
+     Then stdout should be valid URL
+      And "minishift ip" and "minishift console --url" should return same IP address
```

Comment:
@hferentschik Steps suggested by you seems fine, I am just wondering how to get the IP of VM by not using `minishift ip` command. Idea was to extract it from `minishift console --url` and validate them. But it does not make much more sense. 

So maybe steps:

```
And stdout should be the Minishift VM IP
And stdout should be the OpenShift Console URL
```
could be skipped for now. And later when we have a way how to get these values from another souce, we can add them and also add DYNAMIC_VALUE substitution. WDYT?